.github/workflows/rust.yml

```diff
@@ -31,40 +31,39 @@ jobs:
       env:
         RUSTUP_HOME: "/root/.rustup"
         FORCE_COLOR: 1
-
     steps:
-    - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
-    - name: Cache cargo registry
-      uses: actions/cache@v2.1.7
-      with:
-        path: ~/.cargo/registry
-        key: ${{ runner.os }}-v2-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
-
-    - name: Cache cargo index
-      uses: actions/cache@v2.1.7
-      with:
-        path: ~/.cargo/git
-        key: ${{ runner.os }}-v2-cargo-index-${{ hashFiles('**/Cargo.lock') }}
-
-    - name: Cache cargo target dir
-      uses: actions/cache@v2.1.7
-      with:
-        path: contracts/target
-        key: ${{ runner.os }}-v2-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}
-
-    - run: solana-keygen new -o id.json --no-bip39-passphrase
-    - name: Compile typescript client
-      run: |
-       cd ../ts
-       yarn install --frozen-lockfile
-       yarn build
-    - run: yarn install --frozen-lockfile
-    - run: anchor test
-    - run: |
-       cd examples/hello-world
-       yarn install --frozen-lockfile
-       anchor test
-
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: Cache cargo registry
+        uses: actions/cache@v2.1.7
+        with:
+          path: ~/.cargo/registry
+          key: ${{ runner.os }}-v2-cargo-registry-${{ hashFiles('**/Cargo.lock') }}
+  
+      - name: Cache cargo index
+        uses: actions/cache@v2.1.7
```

Comment:
I think the formatting messed this up where it's not updating the change

Reply:
No there's a lot of complications with these workflows.

Anytime there's a:
```
    container:
image: projectserum/build:${{ needs.get_projectserum_version.outputs.projectserum_version }}
```

on a job, then it can't support node20 actions. So all of  these are being left alone.

Reply:
ahh gotcha cool